TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -1962,5 +1968,34 @@ void DefaultRenderOverlay(RenderManager.CameraInfo cameraInfo)
                 OnscreenDisplay.DisplayIdle();
             }
         }
+        public void AddUUIButton() {
+            try {
+                var hotkeys = new UUIHotKeys { ActivationKey = KeybindSettingsBase.ToggleMainMenu.Key };
+                hotkeys.AddInToolKey(KeybindSettingsBase.ToggleTrafficLightTool.Key);
+                hotkeys.AddInToolKey(KeybindSettingsBase.LaneArrowTool.Key);
+                hotkeys.AddInToolKey(KeybindSettingsBase.LaneConnectionsTool.Key, () => Options.laneConnectorEnabled);
```

Comment:
Names on the options are ambiguous
We have switches in TM:PE which enable entire feature
And also we have hotkeys which don't toggle the feature, but they activate tool button
Change names for hotkeys which activate tools from `***Enabled` to for example `Use***Tool`

Reply:
it is out of scope of this PR. you should create an issue.